wiki/Tournaments/SOFT/6/en.md

```diff
@@ -32,7 +32,7 @@ The **Springtime osu!mania Free-for-all Tournament 6** (***SOFT 6***) is a 1v1 c
 
 ## Prizes
 
-The Springtime osu!mania Free-for-all Tournament 6 will offer a $250 minimum cash prize pool!
+The Springtime osu!mania Free-for-all Tournament 6 offered a $250 minimum cash prize pool!
```

Comment:
do you know the final amount?

Reply:
I do actually, it ended up being $260. I will edit it and remove the 'minimum', since it was the final number anyway.